tensorflow/python/framework/experimental/math_ops.py

```diff
@@ -30,3 +30,8 @@ def add(a, b, name=None):
 def mat_mul(a, b, name=None):
   ctx = context.get_default()
   return _math_ops.mat_mul(ctx, a, b, name)
+
+
+def neg(a, name=None):
```

Comment:
Could we add a test for this as well? You can look at tests in `unified_api_test.py`.